GitFyle.Core.Api/Services/Foundations/Contributions/ContributionService.Exceptions.cs

```diff
@@ -70,6 +74,31 @@ private async ValueTask<Contribution> TryCatch(ReturningContributionFunction ret
             }
         }
 
+        private async ValueTask<IQueryable<Contribution>> TryCatch(ReturningContributionsFunction returningContributionsFunction)
```

Comment:
```suggestion
        private async ValueTask<IQueryable<Contribution>> TryCatch(
            ReturningContributionsFunction returningContributionsFunction)
```
This line is over 120